host-controller/src/main/java/org/jboss/as/host/controller/HostControllerConfigurationPersister.java

```diff
@@ -82,8 +82,7 @@ public void initializeDomainConfigurationPersister(boolean slave) {
         ConfigurationFile domainConfigurationFile = null;
         if (slave) {
             // in either case of --backup and/or --cached-dc, we persist with the same persister
-            if (environment.isBackupDomainFiles() || environment.isUseCachedDc()) {
-                domainConfigurationFile = getBackupDomainConfigurationFile();
```

Comment:
Why is this removed? AFAICT the only effect of this is null now gets passed to this below:

```
        // Store this back to environment so mgmt api that exposes it can still work
        environment.setDomainConfigurationFile(domainConfigurationFile);
```

Looking at the impact of that being null, it would only be to disable output from HostEnvironmentResourceDefinition.HostEnvironmentReadHandler. The other read of that value in HostProcessReloadHandler won't happen on a slave, so what we pass in here doesn't matter.